jobs/dynatrace-oneagent/templates/pre-start.erb

```diff
@@ -112,9 +112,9 @@ downloadAgent() {
     local installerFile=$2
 
     if curl -h &> /dev/null ; then
-        local downloadCommand="curl -s -S -f --connect-timeout 10 $SSL_INSECURE_CURL ${downloadUrl} -o ${installerFile}"
+        local downloadCommand="curl -s -S -f --connect-timeout 10 $SSL_INSECURE_CURL ${downloadUrl} -o ${installerFile} --header 'x-dynatrace-iorigin:bosh'"
```

Comment:
Since the header is used two times, please put it into a variable.

Something like `headers="x-dynatrace-iorigin:bosh"` and use that with the `--header` argument